src/sas/system/config/config_meta.py

```diff
@@ -42,9 +43,16 @@ def __init__(self):
         self._schema: Dict[str, SchemaElement] = {}
         self._defaults: Dict[str, SchemaElement] = {}
         self._deleted_attributes: List[str] = []
-        self._write_disabled = False
+        self._bad_entries: Dict[str, Any] = {}
```

Comment:
A tad pejorative: Just because it is unused in the current version doesn't mean it is "bad" 8-)  I suggest a more neutral `_unused_entries`. Also, if you allow unused entries to be propagated I'm not sure you need to handle deleted attributes specially.

Reply:
deliberately pejorative.

Reply:
Although there is an argument for allowing different versions to be installed, people shouldn't have a need to use an older version unless we've messed up (I agree this is quite possible). They are bad keys, created from badness.